ExTrip/Controllers/HotelBooking/Hotel/HotelResultViewController.swift

```diff
@@ -0,0 +1,206 @@
+        else {
+            stopLoading()
+            // Show something ...
+            print("error when fetching")
+        }
+    }
+    
+    private func regiterCell() {
+        tableView.register(HotelResultTableViewCell.self,
+                           forCellReuseIdentifier: HotelResultTableViewCell.identifier)
+    }
+    
+    private func setupBinder() {
+        bookingViewModel.hotelsRelatedCity.bind { [weak self] hotels in
+            if let hotels = hotels, hotels.count > 0 {
+                self?.hotels = hotels
+            } else {
+                self?.stopLoading() 
+            }
+        }
+        
+        bookingViewModel.roomAvailible.bind { [weak self] rooms in
+            self?.rooms = rooms
+        }
+    }
+    
+    // MARK: - Setup UI
+    private func setupViews() {
+        view.backgroundColor = .systemBackground
+        view.addSubviews(tableView)
+        view.insertSubview(backgroundImageView, at: 0)
+        tableView.delegate = self
+        tableView.dataSource = self
+        
+        setupConstaintsView()
+    }
+    
+    private func setupConstaintsView() {  
+        backgroundImageView.anchor(top: view.safeAreaLayoutGuide.topAnchor,
+                                   bottom: view.bottomAnchor,
+                                   leading: view.leadingAnchor,
+                                   trailing: view.trailingAnchor)
+        
+        tableView.anchor(top: view.safeAreaLayoutGuide.topAnchor,
+                         bottom: view.bottomAnchor,
+                         leading: view.leadingAnchor,
+                         trailing: view.trailingAnchor)
+
+    }
+    
+}
+
+// MARK: - UITableViewDelegate, UITableViewDataSource
+extension HotelResultViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return hotels?.count ?? 0
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: HotelResultTableViewCell.identifier, for: indexPath) as? HotelResultTableViewCell else { return UITableViewCell() }
+        if let hotel = hotels?[indexPath.item] {
+            
+            var room: [RoomModel] = []
+            if let rooms = rooms {
+                room = rooms.filter { $0.hotelID == hotel.id }
+            } 
+            
+            cell.numberRoomAvailable = room.count
+            
+            cell.setupDataInfoHotelBooking(image: hotel.thumbnail,
+                                           title: hotel.name,
+                                           star: hotel.star,
+                                           service: hotel.rating,
+                                           review: hotel.review,
+                                           address: hotel.address, 
+                                           soldOutPrice: hotel.price)
```

Comment:
should pass hotel as a parameter. look clean
```suggestion
            cell.setupDataInfoHotelBooking(hotel: hotel)
```